src/widgets/import_widget.py

```diff
@@ -1,12 +1,146 @@
+
+        # columns group
+        self.columns_group = QGroupBox(self.frame)
+        self.columns_group.setTitle("Columns:")
+        self.columns_group.setGeometry(QRect(360, 30, 400, 100))
+        self.columns_grid = QGridLayout()
+        self.columns_group.setLayout(self.columns_grid)
+
+    # set titles to box
+    def set_available_tables(self):
+        names = self.engine.get_table_names_from_database()
+        for name in names:
+            self.database_box.addItem(name)
+
+    # enable buttons after load data
+    def set_options(self):
+        self.save_button.setEnabled(True)
+        if self.engine.is_data_big():
+            self.warning_label.setText("This file is too big.\nYou must save it in database!")
+        else:
+            self.not_save_button.setEnabled(True)
+
+    # clear import widget from loaded data
+    def clear_widgets(self):
+        self.error_label.clear()
+        self.warning_label.clear()
+        for i in reversed(range(self.columns_grid.count())):
+            self.columns_grid.itemAt(i).widget().deleteLater()
+
+    # draw columns and checkbox to choose them
+    def set_columns_grid(self):
+        columns = self.engine.get_columns()
+        col = max(len(columns) // 11 + 1, 2)
+        rows = (len(columns) - 1) // col + 1
+        self.columns_group.setFixedHeight(min(rows*50, 450))
+        positions = [(i, j) for i in range(rows) for j in range(col)]
+        for name, position in zip(columns, positions):
+            checkbox = QCheckBox(name)
+            checkbox.setChecked(True)
+            self.columns_grid.addWidget(checkbox, *position)
+
+    # get chose columns
+    def get_checked_columns(self) -> List[str]:
+        columns = []
+        for i in range(self.columns_grid.count()):
+            if self.columns_grid.itemAt(i).widget().isChecked():
+                columns.append(self.columns_grid.itemAt(i).widget().text())
+        return columns
+
+    def click_listener(self, button_type: str):
+        if button_type == 'load_file':
+            self.error_label.setText("Loading ...")
+            filepath = self.filepath_line.text()
+            result = self.engine.load_data_from_file(filepath)
+            if result:
+                self.error_label.setText(result)
+                return
+            self.clear_widgets()
+            self.set_options()
+            self.set_columns_grid()
+        elif button_type == 'load_database':
+            self.error_label.setText("Loading ...")
+            document_name = self.database_box.currentText()
+            result = self.engine.load_data_from_database(document_name)
+            if result:
+                self.error_label.setText(result)
+                return
+            self.clear_widgets()
+            self.set_options()
+            self.set_columns_grid()
+        elif button_type == 'reject_data':
+            self.clear_widgets()
+            self.engine.clear_import()
```

Comment:
```suggestion
        elif button_type == 'reject_data':
            self.clear_widgets()
            self.engine.clear_import()
            self.save_button.setEnabled(False)
            self.not_save_button.setEnabled(False)
```

With this is connected some bug. After sequence: input the path -> Load button click -> Reject this data, other two buttons under Options group stays enabled. Clicking any of them crashes the program. Proposed solution above